src/documents/tests/test_api.py

```diff
@@ -1844,6 +1846,45 @@ def test_get_comments_no_doc(self):
         )
         self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
 
+    def test_tag_unique_name_and_owner(self):
+        user1 = User.objects.create_user(username="test1")
+        user2 = User.objects.create_user(username="test2")
+
+        self.client.post("/api/tags/", {"name": "tag 1"}, format="json")
+
+        with transaction.atomic():
+            with self.assertRaises(IntegrityError):
```

Comment:
Shouldn't the API return a standard error code, such as HTTP_400_BAD_REQUEST, not raise an exception?

Reply:
Doh, very good point. I'll sort that out

Reply:
Ah, encode/django-rest-framework/issues/7173

Reply:
This should be sorted, lmk thoughts